library/src/main/java/com/pokegoapi/api/PokemonGo.java

```diff
@@ -18,9 +18,13 @@
 import POGOProtos.Networking.Envelopes.RequestEnvelopeOuterClass.RequestEnvelope.AuthInfo;
 import POGOProtos.Networking.Envelopes.SignatureOuterClass;
 import POGOProtos.Networking.Envelopes.Unknown6OuterClass;
+import POGOProtos.Networking.Requests.RequestTypeOuterClass.RequestType;
```

Comment:
Are you sure this imporrs are needed, because you doesn't changed the code in this file.